crates/miden-objects/src/note/recipient.rs

```diff
@@ -73,7 +73,7 @@ impl NoteRecipient {
     /// - INPUTS_COMMITMENT is the commitment of the note inputs
     /// - SCRIPT_ROOT is the commitment of the note script (i.e., the script's MAST root)
     /// - SERIAL_NUMBER is the recipient's serial number
-    pub fn format_for_advice(&self) -> Vec<Felt> {
+    pub fn to_elements(&self) -> Vec<Felt> {
```

Comment:
If we go through with the suggestion from the previous comment, we won't be able to use this `to_elements()` to reduce the recipient to a sequential list of elements. From what I can tell though, this function is used in only one place `add_output_note_recipient()` and so we could probably move the logic of populating the recipient advice map entries there without too much effort.